app/presenters/call_for_evidence_presenter.rb

```diff
@@ -0,0 +1,122 @@
+class CallForEvidencePresenter < ContentItemPresenter
+  include ContentItem::Body
+  include ContentItem::Metadata
+  include ContentItem::NationalApplicability
+  include ContentItem::Political
+  include ContentItem::Shareable
+  include ContentItem::TitleAndContext
+  include ContentItem::Attachments
+  include ContentItem::SinglePageNotificationButton
+
+  def opening_date_time
+    content_item["details"]["opening_date"]
+  end
+
+  def closing_date_time
+    content_item["details"]["closing_date"]
+  end
+
+  def opening_date
+    display_date_and_time(opening_date_time)
+  end
+
+  def opening_date_midnight?
+    Time.zone.parse(opening_date_time).strftime("%l:%M%P") == "12:00am"
+  end
+
+  def closing_date
+    display_date_and_time(closing_date_time, rollback_midnight: true)
+  end
+
+  def open?
+    document_type == "open_call_for_evidence"
+  end
+
+  def closed?
+    %w[closed_call_for_evidence call_for_evidence_outcome].include? document_type
+  end
+
+  def unopened?
+    !open? && !closed?
+  end
+
+  def outcome?
+    document_type == "call_for_evidence_outcome"
+  end
+
+  def outcome_detail
+    content_item["details"]["outcome_detail"]
+  end
+
+  def outcome_documents
+    content_item["details"]["outcome_documents"]&.join("")
+  end
```

Comment:
I understand in addressing this https://github.com/alphagov/government-frontend/pull/2828#discussion_r1265224977 this can now return either nil or a string? Can the place where this is used accept both?

Though I think this might just be used for the legacy documents so might be removalable?

Reply:
Yes, there are presence checks on the result of this in the attachments rendering partial. I will look into the possibility of removing the legacy rendering altogether for calls for evidence - I need to understand more about what legacy means in this context